tests/rptest/tests/random_node_operations_test.py

```diff
@@ -120,6 +122,12 @@ def early_exit_hook(self):
 
     def setUp(self):
```

Comment:
:( unfortunate that we hog on 2 nodes for this which for half of the tests are not even used and this tests runs for quite a bit of time too. Especially in CDT, we waste dollars on idle machines. Do you think we can split the test entrypoint so this does not happen?

I believe you just need to split the annotation. I.e.
```
@cluster(num_nodes=10)
@matrix(...)
@cluster(num_nodes=3)
@matrix(...)
```

This is based on my reading of code from some time ago. Didn't actually test it.